### DISCUSSION TITLE
我在php的容器找不到supervisord

### DISCUSSION THREAD

**Discussion by iygwei** [1Panel-dev/1Panel#6260] [Q&A]
我在你们的https://hub.docker.com/layers/1panel/php/8.0.30-fpm/images/sha256-b7b1d214e14babdd2833eb51cc0b240deecc50360fb18c41962fd724b45a4e34?context=explore
这个里看到是有的
![Snipaste_2024-08-27_11-00-33](https://github.com/user-attachments/assets/01ad2258-0067-48c3-9a5a-75d18225700c)
为什么我进入容器里面以后又不见了?在搬生产的生产挺急的

**Comment by wanghe-fit2cloud** [comment]
在 1Panel v2 版本中，创建 PHP 运行环境时，容器内将默认支持 supervisord。
对于 v1 版本，你可以参考论坛文章中的相关教程。

**Reply by iygwei** [reply]
怎么升级v2 我现在是v1.10.13-lts

**Reply by zhengkunwang223** [reply]
> 怎么升级v2 我现在是v1.10.13-lts

https://bbs.fit2cloud.com/t/topic/5201
